specification/dartLangSpec.tex

```diff
@@ -16727,7 +16735,7 @@ \subsection{Identifier Reference}
 
 <DIGIT> ::= `0' .. `9'
 
-<WHITESPACE> ::= (`\\t' | ` ' | <NEWLINE>)+
+<WHITESPACE> ::= (`\\t' | ` ' | <LINE\_BREAK>)+
```

Comment:
I believe we have this already: Line 9153 is

```latex
  \syntax{`\\t'} for tab, equivalent to \syntax{`\\x09'}.
```

and `\\x09` is specified to be the same as `\\u{09}` which is specified to be the Unicode character with code point U+0009. We don't say tab anywhere, but I guess that would be known because it is true for U+0009 with no reference to Dart at all.